projects/storefrontlib/src/cms-structure/seo/seo-meta.service.ts

```diff
@@ -52,6 +55,16 @@ export class SeoMetaService implements OnDestroy {
     }
   }
 
+  /**
+   * Add the canonical Url to the head of the page.
+   *
+   * If the canonical url already exists the link is removed. This is quite
+   * unlikely though, since canonical links are (typically) only added in SSR.
+   */
+  protected set canonicalUrl(url: string) {
+    this.pageMetaLinkService.addCanonicalLink(url);
```

Comment:
What about adding a disabled/enabled flag on the  `PageMetaResolverConfig`, similar to `disabledInCsr`? We could add a property, or introduce an enumeration for each resolver: `DISABLED`, `DISABLED_IN_CSR` and `DISABLED_IN_CSR`.

Reply:
Ok, we agreed on the following: we're dropping the default `canonicalUrl` configuration from the `PageMetaConfig` so that it won't be used unless specifically configured. This will then remove the breaking change and still customers are able to enable the feature right away.